lib/galaxy/tool_util/deps/mulled/invfile.lua

```diff
@@ -45,6 +45,7 @@ if conda_image == '' then
     conda_image = 'continuumio/miniconda3:latest'
```

Comment:
```suggestion
    conda_image = 'quay.io/condaforge/mambaforge:latest'
```

Reply:
When we use `continuumio/miniconda3:latest` and then do `conda install ... -c conda-forge`, then we re-install every package since we switch from `defaults` to `conda-forge`.

Reply:
With `condaforge/mambaforge` we avoid that re-installation and also have `mamba` installed from the get go.
Caveat: `condaforge/mambaforge` explicitly has only `conda-forge` in the channel list, i.e., `default` removed. Personally, I think that's fine/good. However, if that change messes something up on your/someone's end, then we'd have to *always* run `conda config --append channels defaults` in the `PREINSTALL`.

Reply:
Thanks, do we need this to be also a parameter for `mull_targets()` below? Currently this seems only configurable via the `CONDA_IMAGE` environment variable, unlike `DEST_BASE_IMAGE` which can be configured both as environment variable and with the `base_image` parameter.

Reply:
I agree. I only brought up `defaults` removal so you're aware of it and because I don't know in which other contexts `mulled` is used around Galaxy. Generally, I'd endorse/recommend only using `conda-forge`+`bioconda` by now.